conda_forge_tick/update_sources.py

```diff
@@ -553,11 +553,30 @@ def get_url(self, meta_yaml) -> Optional[str]:
 
 class Github(VersionFromFeed):
     name = "Github"
+    version_prefix = None
+
+    def set_version_prefix(self, version: str, split_url: list[str]):
+        self.version_prefix = self.get_version_prefix(version, split_url)
+        if self.version_prefix is None:
+            return
+        logger.debug(f"Found version prefix from url: {self.version_prefix}")
+        self.ver_prefix_remove = [self.version_prefix] + self.ver_prefix_remove
+
+    def get_version_prefix(self, version: str, split_url: list[str]):
+        r = re.compile(rf"^(.*){version}")
+        for split in split_url:
+            match = r.match(split)
+            if match is not None and match.group(1) != "":
+                return match.group(1)
+
+        return None
 
     def get_url(self, meta_yaml) -> Optional[str]:
         if "github.com" not in meta_yaml["url"]:
             return None
         split_url = meta_yaml["url"].lower().split("/")
+        version = meta_yaml["version"]
+        self.set_version_prefix(version, split_url)
```

Comment:
```suggestion
        if "/archive/refs/tags/" in meta_yaml["url"]:
            self.set_version_prefix(version, split_url)
```

Maybe we can do this for now? This would only search for prefixes in the most common case of using a url of the source archive from a GH release.

Reply:
See my other comment. I guess I've added better solution.